oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/profile/analyze/ProfileAnalyzer.java

```diff
@@ -88,7 +88,7 @@ public ProfileAnalyzation analyze(String segmentId, List<ProfileAnalyzeTimeRange
         }).flatMap(Collection::stream).map(ProfileStack::deserialize).distinct().collect(Collectors.toList());
 
         // analyze
-        analyzation.setTrees(analyze(stacks));
+        analyzation.getTrees().addAll(analyze(stacks));
```

Comment:
What is the difference? `addAll(null)` will cause a NPE from my understanding.

Reply:
My fault. I will add an `if` judge before `addAll`